app/assets/javascripts/components/stories/History/HistoryChanges/HistoryChange.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const HistoryChange = ({
+  oldValue,
+  newValue,
+  title
+}) =>
+  <div className="info">
+    <div className="action" data-id="history-change-title">
+      { title }
+    </div>
+    <div className="changes">
+      <div data-id="history-old-value"> { oldValue } </div>
+      <div>
+        <i className="mi md-18">arrow_forward</i>
+      </div>
+      <div data-id="history-new-value"> { newValue } </div>
+    </div>
+  </div>
+
+HistoryChange.defaultProps = {
+  oldValue: '---',
+  newValue: '---'
+};
+
+HistoryChange.propTypes = {
+  oldValue: PropTypes.oneOfType([
+    PropTypes.number,
+    PropTypes.string
```

Comment:
Why can this be either a number or a string?

Reply:
In the most part of the cases is a `string`, like change the description, story type, etc, but when the estimate is changed, the `oldValue` will be a number. Example:
![Screenshot from 2019-12-10 10-02-00](https://user-images.githubusercontent.com/49662698/70531821-7c8baf80-1b34-11ea-9a90-51a82d633552.png)

Estimate is number, story type and updated at is a string.

Reply:
Can we create a shape for that then?